src/main/io/gps.c

```diff
@@ -174,6 +181,12 @@ static const uint8_t ubloxInit[] = {
     0xB5, 0x62, 0x06, 0x01, 0x03, 0x00, 0x01, 0x12, 0x01, 0x1E, 0x67,           // set VELNED MSG rate
 
     0xB5, 0x62, 0x06, 0x08, 0x06, 0x00, 0xC8, 0x00, 0x01, 0x00, 0x01, 0x00, 0xDE, 0x6A,             // set rate to 5Hz (measurement period: 200ms, navigation rate: 1 cycle)
+	// for M8M. set GLONASS on, Galileo on, BeiDou off, Auto Channels
+	0xB5, 0x62, 0x06, 0x3E, 0x3C, 0x00, 0x00, 0x00, 0x20, 0x07, 0x00, 0x08, 0x10, 0x00,
```

Comment:
Wrong indentation. Please use 4 spaces instead of 1 tab as per Betaflight coding standards.

Reply:
Wasn't aware of this rule. Most external editors reduce spaces to tabs...